code/bookmarklet_builder.js

```diff
@@ -1,23 +1,27 @@
 (function() {
   // Live version
-  let HOST = 'https://r.keymanweb.com';
+  var HOST = 'https://r.keymanweb.com';
 
   // // Debug / dev version
-  // let HOST = 'http://r.keymanweb.com.local';
+  // var HOST = 'http://r.keymanweb.com.local';
 
   function construct_bookmarklet_stylesheet() {
-    let style = document.createElement('style');
+    var style = document.createElement('style');
 
-    let imgHref = `${HOST}/code/keymanweb-icon-16.png`;
+    var imgHref = `${HOST}/code/keymanweb-icon-16.png`;
+
+    var styleSpec = `
+    /* Easier to override when less specific */
```

Comment:
This is a process comment and doesn't really make sense to future readers? Easier than what?

Reply:
See https://github.com/keymanapp/keyman.com/pull/346/commits/96b8b44a22736e5a0d4e90531881c3007017fccc - I needed a way to override the font-size on the bookmarklet there to keep their text the same size as before on the corresponding page.

`.bookmarklet a` is more specific than `.bookmarklet` (inherited by child elements).  As the page's CSS is linked before the bookmarklet's CSS (at least, in the existing, not yet changed design), my understanding is that the bookmarklet CSS will override any pre-set font size unless I can apply it to something "more specific".

I'm not sure what the best technical language is for saying _that_ would be.

Reply:
I see. There are many ways to skin that cat in the keyman.com CSS, from the big hammer `!important` to the slightly smaller hammer `html .keyman-bookmarklet a`, and so forth. What you've done here also works, but the comment confused me!